### PR TITLE
Service + proxy run locally

### PR DESCRIPTION
**Pull request by paloma-vel**

### CODE REVIEW

File: client/src/components/App.jsx [hunk 1]
```diff
@@ -90,10 +88,6 @@ class App extends React.Component {
 
     axios.get(`/api/watches/${wid}/reviews`)
```
Comment (milesmanning) [inline client/src/components/App.jsx:89]: Take a second look at this syntax?

File: database/postgres/index.js [hunk 2]
```diff
@@ -1,52 +1,66 @@
...
+// READ
+const getReviewsById = (w_id, callback) => {
+  const queryString = `select * from reviews where w_id = ${w_id}`;
+  pool.query(queryString, (err, results) => {
+    if (err) {
+      callback(error);
+    } else {
+      const data = results.rows;
+      callback(data);
+    }
```
Comment (milesmanning) [inline database/postgres/index.js:43]: Does this callback not take in two parameters? Typically a success callback would look like: 'callback(null, data)'.

File: database/postgres/index.js [hunk 3]
```diff
@@ -1,52 +1,66 @@
...
+// ----- Database Functions ----- //
+
+// CREATE
+const addReview = (w_id, callback) => {
+  const queryString = `insert into reviews(w_id, reviewer, stars, date_posted, review_header, review_body, upvotes, downvotes)`;
+  pool.query(queryString, (err, results) => {
+    if (err) {
+      callback(error);
+    } else {
+      callback(results);
```
Comment (milesmanning) [inline database/postgres/index.js:28]: See other comment

**Review by milesmanning — COMMENTED** [review]
Check these out real quick and we'll merge right after

File: database/postgres/index.js (hunk 2 above)
Comment (megcblake) [reply database/postgres/index.js:43]: Per our conversation, go with what works! You'll have plenty of time afterward to refactor and clean up your code. Approved!

**Pull request merged**